secretstores/azure/keyvault/keyvault.go

```diff
@@ -56,6 +56,7 @@ func (k *keyvaultSecretStore) Init(metadata secretstores.Metadata) error {
 	}
 
 	k.vaultName = settings.Values[componentVaultName]
+	k.vaultDNSSuffix = settings.AzureEnvironment.KeyVaultDNSSuffix
```

Comment:
Please, keep the default behavior so it is backwards compatible.

Reply:
Yes this will be backwards-compatible. Unless someone manually opts in to use a different Azure cloud, this will match the same URL as before.